Deploy/stacks/db/blazegraph/docker-entrypoint.sh

```diff
@@ -4,13 +4,19 @@
 
 
 # Update the contents of WEB-INF in the blazegraph war file
-pushd /tmp/blazegraph
+pushd /tmp/blazegraph || exit
+
+if [[ -n $BLAZEGRAPH_USER && -n $BLAZEGRAPH_PASSWORD_FILE && -f $BLAZEGRAPH_PASSWORD_FILE ]]; then
+    # Set username and password in the tomcat-users.xml using docker secret values
+    sed -i -e "s/BLAZEGRAPH_USER/$BLAZEGRAPH_USER/" -e "s/BLAZEGRAPH_PASSWORD/$(cat $BLAZEGRAPH_PASSWORD_FILE)/" /usr/local/tomcat/conf/tomcat-users.xml
+else
+    # Remove "security-constraint" related blocks
+    sed -i '6,24d' WEB-INF/web.xml
```

Comment:
This is nitpicking, but maybe removing content matching tags of the "security-constraint" related blocks is more robust against line number changes?

Reply:
Good point, improved in commit e1c577a.